src/pluginHooks.js

```diff
@@ -731,6 +745,19 @@ const REGISTERED_HOOKS = [
    */
   'beforeOnCellMouseDown',
 
+  /**
+   * Fired after the user clicked a cell, but before all the calculations related with it.
+   *
+   * @event Hooks#beforeOnCellContextMenu
+   * @since 4.1.0
+   * @param {Event} event The `contextmenu` event object.
+   * @param {CellCoords} coords Cell coords object containing the visual coordinates of the clicked cell.
+   * @param {HTMLTableCellElement} TD TD element.
+   * @param {Object} controller An object with keys `row`, `column` and `cells` which contains boolean values. This
+   *                            object allows or disallows changing the selection for the particular axies.
```

Comment:
The comment says that there is the fourth argument but in fact, the hook is triggered only with three arguments (https://github.com/handsontable/handsontable/pull/5221/files#diff-24033f033992c272cc008ff988b20046R274). The fourth argument should be removed?